processout-sdk/src/main/java/com/processout/processout_sdk/AuthorizationResult.java

```diff
@@ -0,0 +1,25 @@
+package com.processout.processout_sdk;
+
+import com.google.gson.annotations.SerializedName;
+import com.processout.processout_sdk.CustomerAction;
+
+class AuthorizationResult {
+    @SerializedName("customer_action")
+    private CustomerAction customerAction;
```

Comment:
Does this support cases where the data sent in the JSON is null?

Reply:
Yep, the object will be null